tier4_localization_msgs/srv/InitializeLocalization.srv

```diff
@@ -0,0 +1,12 @@
+uint8 AUTO = 0
+uint8 DIRECT = 1
+
+geometry_msgs/PoseWithCovarianceStamped[<=1] pose_with_covariance
+uint8 method
+---
+uint16 ERROR_UNSAFE = 1
+uint16 ERROR_GNSS_SUPPORT = 2
+uint16 ERROR_GNSS = 3
+uint16 ERROR_ESTIMATION = 4
+
+autoware_common_msgs/ResponseStatus status
```

Comment:
I know this is mainly a copy from adapi_msgs, but I suggest to add comments for each variables to make it more user friendly, for example like in [pointcloud2 msg](https://docs.ros.org/en/melodic/api/sensor_msgs/html/msg/PointCloud2.html). Something like the following might work:

```
# constants for specifying initialization method
uint8 AUTO = 0
uint8 DIRECT = 1

# input pose for initialization
geometry_msgs/PoseWithCovarianceStamped[<=1] pose_with_covariance

# Variable to set initialization method
# AUTO: The initial position is automatically estimated with localization algorithm. 
#              The input pose will be used as an initial guess if provided. 
# DIRECT: The initial position is set directly by the input pose without going through localization algorithm.
uint8 method
---
# ERROR CODES used in response status in case of failure
uint16 ERROR_UNSAFE = 1
uint16 ERROR_GNSS_SUPPORT = 2
uint16 ERROR_GNSS = 3
uint16 ERROR_ESTIMATION = 4

autoware_common_msgs/ResponseStatus status
```

Reply:
Otherwise, you can create a README for this service and put a link to that document as a comment.

Reply:
Thanks for your suggestion!
I have added your comments https://github.com/tier4/tier4_autoware_msgs/pull/121/commits/49ccda11e3199b02863ecc7d59309fd1e3a4a4a2